CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+* A `preview` property to the `Article` object.
```

Comment:
This is not overly helpful, I'm afraid. Why did we add a preview property to the object? What's the underlying reason for this change?

Reply:
```suggestion
* A fallback `Note` for `Article` objects to improve previews on services that don't support Articles yet.
```

Reply:
Something like that?